charts/redpanda/values.yaml

```diff
@@ -87,6 +87,19 @@ license_secret_ref: {}
   # secret_name: my-secret
   # secret_key: key-where-license-is-stored
 
+# -- Audit logging for a redpanda cluster, must enable sasl and have one kafka listener supporting sasl
+#    for auditing to work
+auditLog:
+  # -- Enable or disable audit logging, for production clusters we suggest you enable.
+  #    however, this will only work if you also enable sasl and a listener with sasl enabled.
+  enabled: true
+  # -- Kafka external listener name, note that it must have `authenticationMethod` set to sasl
+  listener: default
+  # --  Integer value defining the number of partitions used by a newly created audit topic
+  partitions: 12
+  #
```

Comment:
would also suggest adding in:
* `audit_enabled_event_types: ["management","authenticate","admin"]`
* `audit_excluded_topics: []`
* `audit_excluded_principals: []`

Reasonings:

* `audit_enabled_event_types` is probably the 'hottest' thing a customer would want to modify, this controls the content of the audit log and what operations are audited
* `audit_excluded_topics` maybe not as popular but I think this provides a hint to the customer that they can excluded certain topics from auditing that would reduce the amount of audit messages in the topic
* `audit_excluded_principals` along the same vein as `audit_excluded_topics` but probably more useful.  They may assign a specific user to be the auditor (the principal responsible for collecting audit logs).  Regardless of the value of `audit_enabled_event_types`, any access to the audit log is audited, except by principals listed in this list.  So the idea would be that they have an auditor user with ACLS that permit them to only read from the audit log topic.  Without adding that principal to this list, every access to that topic by that user is audited.  By adding the user to this list, those events are not generated, reducing 'junk' in the audit log